images/pod/Dockerfile

```diff
@@ -1,16 +1,22 @@
 #
-# This is the official OpenShift Origin pod infrastructure image. It will stay running
+# This is the official OpenShift pod infrastructure image. It will stay running
 # until terminated by a signal and is the heart of each running pod. It holds on to
 # the network and IPC namespaces as containers come and go during the lifetime of the
 # pod.
 #
 # The standard name for this image is openshift/origin-pod
 #
-FROM scratch
+FROM openshift/origin-source
```

Comment:
I guess this change is going to grow the origin-pod image by a fair amount since origin-source is based on centos. @smarterclayton @stevekuznetsov thoughts?